internal/tools/clickhouse_create_cursors_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE cursors (
+    `chain_id` UInt256,
+    `cursor_type` String,
+    `cursor_value` String,
+    `insert_timestamp` DateTime DEFAULT now(),
+) ENGINE = ReplacingMergeTree(insert_timestamp)
+ORDER BY (chain_id, cursor_type);
```

Comment:
this means the data won't be chronological (latest to oldest)?

Reply:
It's a single table row that will be overwritten after a successful scan